### DISCUSSION TITLE
Initialize Sentry in development

### DISCUSSION THREAD

**Discussion by dcramer** [epicweb-dev/epic-stack#650] [Ideas]
Couple of core reasons:

1. It ensures same behavior (code paths) for both prod and dev
2. It'll enable adoption of https://spotlightjs.com

**Comment by kentcdodds** [comment]
Very interesting! I remember seeing this at some point but never looked into it.

I'll need to play around with it before I decide to have it built into the template. If anyone wants to encourage me to play with it sooner, [an example](https://github.com/epicweb-dev/epic-stack/blob/main/docs/examples.md) is the way to do that.
